src/index.mjs

```diff
@@ -0,0 +1,209 @@
+      entry: [
+        { "@key": "host", $: DATA_STORE_CREDENTIALS.host },
+        { "@key": "port", $: DATA_STORE_CREDENTIALS.port },
+        { "@key": "database", $: DATA_STORE_CREDENTIALS.database },
+        { "@key": "user", $: DATA_STORE_CREDENTIALS.user },
+        { "@key": "passwd", $: DATA_STORE_CREDENTIALS.password },
+        { "@key": "dbtype", $: DATA_STORE_CREDENTIALS.dbtype },
+        { "@key": "schema", $: DATA_STORE_CREDENTIALS.schema },
+        { "@key": "Evictor run periodicity", $: "300" },
+        { "@key": "Max open prepared statements", $: "50" },
+        { "@key": "encode functions", $: "true" },
+        { "@key": "Batch insert size", $: "1" },
+        { "@key": "preparedStatements", $: "false" },
+        { "@key": "Loose bbox", $: "true" },
+        { "@key": "SSL mode", $: DATA_STORE_CREDENTIALS.ssl },
+        { "@key": "Estimated extends", $: "true" },
+        { "@key": "fetch size", $: "1000" },
+        { "@key": "Expose primary keys", $: "false" },
+        { "@key": "validate connections", $: "true" },
+        { "@key": "Support on the fly geometry simplification", $: "true" },
+        { "@key": "Connection timeout", $: "20" },
+        { "@key": "create database", $: "false" },
+        {
+          "@key": "Method used to simplify geometries",
+          $: "PRESERVETOPOLOGY",
+        },
+        { "@key": "min connections", $: "1" },
+        { "@key": "max connections", $: "10" },
+        { "@key": "Evictor tests per run", $: "3" },
+        { "@key": "Test while idle", $: "true" },
+        { "@key": "Max connection idle time", $: "300" },
+      ],
+    },
+  },
+};
+
+const createDataStoreResp = await zx.fetch(DATA_STORE_API_URL, {
+  method: "POST",
+  body: JSON.stringify(dataStoreCreationBody),
+
+  headers: {
+    Authorization: "Basic " + btoa(GEOSERVER_USER + ":" + GEOSERVER_PASS),
+    "Content-Type": "application/json",
+  },
+});
+console.log(await createDataStoreResp.text());
+assertEqual(createDataStoreResp.status, 201);
+console.log(
+  `3. Complete creation data-store ${DATA_STORE_NAME} with status code: ${createWorkspaceResp.status}\n`
+);
+
+await zx.sleep(1000);
+
+/**
+ * 4. This stage will send api request to create wfs layer feature
+ */
+
+const layerBody = JSON.parse(fs.readFileSync(LAYER_BODY_JSON));
+layerBody["featureType"]["maxFeatures"] = MAX_FEATURES;
+layerBody["featureType"]["numDecimals"] = NUM_DECIMALS;
+layerBody["featureType"]["title"] = LAYER_TITLE_NAME;
+
+const createLayerResp = await zx.fetch(FEATURE_TYPES_API_URL, {
+  method: "POST",
+  body: JSON.stringify(layerBody),
+  headers: {
+    Authorization: "Basic " + btoa(GEOSERVER_USER + ":" + GEOSERVER_PASS),
+    "Content-Type": "application/json",
+  },
+});
+
+console.log(await createLayerResp.text());
+assertEqual(createLayerResp.status, 201);
+console.log(
+  `4. Complete creation layer ${LAYER_TITLE_NAME} with status code: ${createWorkspaceResp.status}\n`
+);
+
+setInterval(() => {}, 1000000);
```

Comment:
what happens to this side card when this code finishes?
will it restart? 
if so, is deleting and setting the config while service is running is best practice?

Reply:
it enter to infinite interval to keep a live and release the readiness for geoserver service